Code.gs

```diff
@@ -57,10 +65,14 @@ function GenerateForms() {
 
       // add manager as editor
       associateForm.addEditor(managerEmail);
+      if (viewerEmail != managerEmail && viewerEmail != "") {
```

Comment:
goal of this one?

Reply:
@yanjoseph can you follow up on this one?

Reply:
The idea was a manager might send the forms on behalf of other managers (and view the feedback forms as well), but as he might have some directs on that list as well (in which case the viewer == the manager creating the reports) adding viewer is redundant.